blockchain/ethereum/uniswap/v2/core compiler/e0f819d4-ec36-45a0-a772-252e08986fdd.json

```diff
@@ -5,17 +5,19 @@
       "6847d4a5-3a0a-429b-bf8f-410f4da7e1ab": {
         "type": "any",
         "name": "start",
-        "description": ""
+        "description": "Receives a signal that triggers the compilation of the Uniswap V2 Core code."
```

Comment:
Optional: 
We may consider referring to these contracts as Factory (and its counterpart as Router).
(Maybe just adding in parentheses after core to keep consistency with the nodes' names)... Node names unfortunately cannot be be altered because they are featured in the video.

Top level names could have been originally chosen after their published functionality (i.e. factory and router), rather than adopting the top level category names from Uniswap repo (core and periphery). /my fault!/